src/puzzles/day_8.rs

```diff
@@ -0,0 +1,136 @@
+use super::input;
+
+use self::computation::{Instruction, Computer, Termination};
+
+pub fn part_1() -> i32 {
+    let instructions: Vec<Instruction> = input::read_lines(8)
+        .iter()
+        .map(|line| Instruction::from(line).expect("Invalid instruction in input file src/inputs/day_8.txt. Get original file from https://adventofcode.com/2020/day/8/input"))
+        .collect();
```

Comment:
Refactor this parsing _(it is used by both `part_1` and `part_2`)_ into a function like `parse_program()`.